utils/post.js

```diff
@@ -341,7 +341,17 @@ const modifyNewItemAnonymity = (newItem, isBlurredPost = false) => {
   return newItem;
 };
 
+const isUserFollowAuthor = async (newItem, followingUsers) => {
+  if (newItem?.actor?.id) {
+    newItem.is_following_target = followingUsers.includes(newItem?.actor?.id);
+  }
+
+  return newItem;
+};
+
 const modifyFeedIsFollowingTarget = async (newItem, userId) => {
+  // TODO: Need to modify this function
+  // Get user list of following user, then check if the target user is in the list
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a TODO comment indicating that the function `modifyFeedIsFollowingTarget` needs to be modified. It's important to address these TODOs as they can indicate incomplete or temporary code. If the modification isn't necessary anymore, consider removing the comment to avoid confusion.
```diff
-  // TODO: Need to modify this function
-  // Get user list of following user, then check if the target user is in the list
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->